app/src/main/java/org/dicekeys/app/fragments/backup/BackupFragment.kt

```diff
@@ -58,11 +69,15 @@ class BackupFragment: AppFragment<BackupFragmentBinding>(R.layout.backup_fragmen
         super.onViewCreated(view, savedInstanceState)
 
         // Guard: If DiceKey is not available, return
-        repository.get(args.diceKeyId)?.also {
-            diceKey = it as DiceKey<Face>
-        } ?: run {
-            findNavController().popBackStack()
-            return
+        args.diceKeyId?.also {
+            repository.get(it)?.also {
```

Comment:
Another spot where it would be less confusing either if the function name implied what was returned or if `it` were replaced by something explicit.